src/Tag/Base/MediaTag.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag\Base;
+
+use InvalidArgumentException;
+use Stringable;
+use Yiisoft\Html\Tag\Media\Track;
+use Yiisoft\Html\Tag\Source;
+
+abstract class MediaTag extends NormalTag
```

Comment:
We can finalize all methods, since they should not be overridden in the child classes.

Reply:
But `NormalTag` already abstract